content/components/action-bar.mdx

```diff
@@ -223,10 +278,16 @@ An action bar has an ARIA role of [`toolbar`](https://developer.mozilla.org/en-U
 | `End`              | Moves focus to the **last** button.                                                                                                                                                                                                                                                                                                                    |
 | `Enter` or `Space` | Triggers the button **action**.                                                                                                                                                                                                                                                                                                                        |
 
+The `toolbar` role should be overridden if you're using an action bar in a context where the behavior of left arrow, right arrow, "Home", and "End" keys conflict with their behavior in a `toolbar`. For example, as list item actions within a list where focus may be moved between items using arrow, "Home", and "End" keys.
+
+In cases where we don't use the `toolbar` role, no role is applied and each buttons has one tab stop (their default keyboard navigation behavior).
+
 <!-- TODO: Add video -->
 
 ### Touch targets
```

Comment:
The PVC implementation doesn't follow this guidance. An issue has been created: https://github.com/github/primer/issues/3078